docs/tools/offline/cli-test-debug.md

```diff
@@ -99,6 +143,12 @@ You can specify to only **build** the tests by using the `--compile` option:
 $ mbed test -m K64F -t GCC_ARM --compile
 ```
 
+In case of icetea only the test applications are built
```

Comment:
Please phrase sentences preceding code snippets so you can end them with : 

For icetea, only the test applications are built:

```
your code here
```